src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -59,8 +62,12 @@ public HttpResponseMessage Invoke(string name, [FromBody] FunctionInvocation inv
         [Route("admin/functions/{name}/status")]
         public FunctionStatus GetFunctionStatus(string name)
         {
-            FunctionStatus status = new FunctionStatus();
             Collection<string> functionErrors = null;
+            FunctionDescriptor function = _scriptHostManager.Instance.Functions.FirstOrDefault(p => p.Name.ToLowerInvariant() == name.ToLowerInvariant());
+            FunctionStatus status = new FunctionStatus
+            {
+                Metadata = function?.Metadata
```

Comment:
What bits of the metadata do you actually need for CLI? Wondering if instead of shoving this into this status api, we shouldn't instead have a new admin/functions/{name}/metadata api? This API is really about returning status only, and we shouldn't return any irrelevant info I think.